apps/web/components/defined-term/DefinedTerm.tsx

```diff
@@ -14,23 +16,32 @@ export default function DefinedTerm({
   ...props
 }: DefinedTermProps) {
   // TODO: Add glossary term functionality
-  const mockGlossary = {
-    default: { tooltipContent: "Default Tooltip" },
-  };
 
-  return (
+  const button = (
+    <Button
+      variant="glossary"
+      data-term={term}
+      data-testid={`${testid}-${term}`}
+      {...props}
+    >
+      {children}
+    </Button>
+  );
+
+  const tooltipButton = (
     <Tooltip
-      tooltipContent={mockGlossary.default.tooltipContent}
-      triggerContent={
-        <Button
-          variant="glossary"
-          data-term={term}
-          data-testid={`${testid}-${term}`}
-          {...props}
-        >
-          {children}
-        </Button>
-      }
+      tooltipContent={GLOSSARY_TERMS.default.tooltipContent}
+      triggerContent={button}
     ></Tooltip>
   );
+
+  // TODO: Matt or Nicholas, add correct section for customHandler
```

Comment:
Should we link the ticket this should be addressed in?

Reply:
Idts, I'm not sure exactly which ticket it will be but it will definitely be covered when the terms are added and the data structure is implemented.